entrypoint.sh

```diff
@@ -84,7 +84,7 @@ exit_code=0
 if [ "$SHELLCHECK_DISABLE" != "1" ]; then
 	echo -e "Validating shell scripts files using shellcheck\n"
 	# shellcheck disable=SC2086
-	shellcheck_error="$(shellcheck $sh_files)"
+	shellcheck_error="$(cat <(shellcheck $sh_files))"
 	shellcheck_code="$?"
```

Comment:
ah, good point. It won't. Let me take a look

Reply:
Done.

I don't think the `while read -r` will be fully POSIX compliant but as long as this action runs it within a bash script you should be good.

Conformed both on bash 5 (_I upgraded since my last push_)

Reply:
I didn't know about PIPESTATUS.. its made my day worth :)

Reply:
let ship it

Reply:
https://github.com/luizm/action-sh-checker/releases/tag/v0.1.9

Thank you so much @tgallacher